packages/terra-icon/CHANGELOG.md

```diff
@@ -2,683 +2,710 @@
 
 ## Unreleased
 
+- Changed
+  - Updated to `one-cerner-style-icons` v1.44.0
+  - Updated SVG migration script `writeSvg,js` to include a newline character at the end of each SVG file
+  - Updated example generation scripts to point to `terra-core-docs` instead of within the `terra-icons` package
+
 ## 3.46.1 - (January 4, 2022)
 
-* Changed
-  * Migrate docs to terra-core-docs.
+- Changed
+  - Migrate docs to terra-core-docs.
```

Comment:
Why are these being changed from `*` to `-`?

Reply:
I've seen this happen because I had Prettier turned on and it "fixes" quotes and lists in markup. I never figured out how to change that config, so I ended up turning Prettier off.

Reply:
That's what happened. Prettier was to blame.